src/sku/molecule/domain/src/snapshots/molecule_dataset_snapshots.rs

```diff
@@ -98,6 +122,11 @@ impl MoleculeDatasetSnapshots {
         const COLUMN_NAME_TAGS: &str = "tags";
         const COLUMN_NAME_VERSION: &str = "version";
 
+        let alias = odf::DatasetAlias::new(
+            Some(molecule_account_name),
+            odf::DatasetName::new_unchecked("data-room"),
```

Comment:
Constant?

Reply:
Used only in this place, but no problem to add

Done